akka-stream-tests/src/test/scala/akka/stream/actor/ActorPublisherSpec.scala

```diff
@@ -139,7 +139,7 @@ class ActorPublisherSpec extends AkkaSpec(ActorPublisherSpec.config) with Implic
       val probe = TestProbe()
       val ref = system.actorOf(testPublisherProps(probe.ref))
       val p = ActorPublisher[String](ref)
-      val s = StreamTestKit.SubscriberProbe[String]()
+      val s = TestSubscriber.probe[String]()
       p.subscribe(s)
       val sub = s.expectSubscription
```

Comment:
Will add auto style subscriber with one subscription tracking and sugar methods.